product_model_viewer/views/product_views.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="product_template_form_view_3d" model="ir.ui.view">
+        <field name="name">product_template_form_view_3d</field>
+        <field name="model">product.template</field>
+        <field name="inherit_id" ref="product.product_template_form_view"/>
+        <field name="arch" type="xml">
+            <xpath expr="//notebook" position="inside">
+                <page string="3D model">
```

Comment:
Congratulations for this really nice module

Completely expressive like that, I think (only my opinion)
```suggestion
                <page string="3D">
```
When you have many tabs, shorter ones are good ;-)

Reply:
Thanks, done